Clarify hass availability and usage for Home Assistant Core users

erffrfez opened the pull request:
add text as per issue #20086  (https://github.com/home-assistant/home-assistant.io/issues/20086).

also change bash to text in the markdown to remove the colour highlighting from 'exit', 'file' etc

<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
Sorry - deleted these lines were in here by mistake
-->
This is to address the comment in the issue https://github.com/home-assistant/home-assistant.io/issues/20086
While here I changed the markdown to remove un-needed, and slightly distracting colour highlighting (change ```bash to ```text)


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: addresses                       #20086

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **Documentation**
	- Updated documentation for the `hass` command-line tool
	- Clarified tool availability and usage for Home Assistant Core users
	- Changed formatting of command usage instructions from `bash` to `text`
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

erffrfez committed:
Update hass.markdown

add text as per issue #20086  (https://github.com/home-assistant/home-assistant.io/issues/20086).

also change bash to text in the markdown to remove the colour highlighting from 'exit', 'file' etc

frenck commented on source/_docs/tools/hass.markdown:
```diff
@@ -3,10 +3,10 @@ title: "hass"
 ---
 
-The command-line part of Home Assistant is `hass`.
+The command-line part of Home Assistant is `hass`.\
```
```suggestion
The command-line part of Home Assistant is `hass`.

```

frenck commented on source/_docs/tools/hass.markdown:
```diff
@@ -3,10 +3,10 @@ title: "hass"
 
-The command-line part of Home Assistant is `hass`.
+The command-line part of Home Assistant is `hass`.\
+This tool is only available to Home Assistant Core users, is started from the command line on the computer running Home Assistant Core (accessed perhaps via ssh) and is not the same as Home Assistant CLI (aka hass-cli). [link](https://www.home-assistant.io/blog/2019/02/04/introducing-home-assistant-cli/)
```
Small wording adjustment.

Removed the reference of hass-cli, which is no longer part of the Home Assistant project and is a third-party tooling.

```suggestion
This tool is only available to users of the Home Assistant Core installation method. It is started from the command line on the computer running Home Assistant Core (accessed perhaps via SSH).
```

erffrfez replied:
In relative authority on what this thing should say, I'm so far down the list so as to not count!

frenck replied:
I do not fully grasp what you mean by that response. That might be a language gap on my end. Could you clarify your wording differently?

erffrfez committed:
Merge branch 'current' into patch-5

erffrfez committed:
Update source/_docs/tools/hass.markdown

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

erffrfez committed:
Update source/_docs/tools/hass.markdown

I think all I need to do here is 'Commit suggestion' and then it will happen. I have no authority to disagree with the suggested change!

Co-authored-by: Franck Nijhof <frenck@frenck.nl>

erffrfez committed:
Merge branch 'current' into patch-5
